canonical-web-and-design/maas-ui
# feat(ui): build filesystems list in machine storage

## Done

- Built filesystems list in machine storage tab

## QA

### MAAS deployment

To run this branch you will need access to one of the following MAAS deployments:

- [Karura](/HACKING.md#karura)
- [Bolla](/HACKING.md#bolla)
- [A development MAAS](/HACKING.md#development-deployment)
- [A local snap MAAS](/HACKING.md#snap-deployment) (this will not usually have machines)

### Running the branch

You can run this branch by:

- Serving with [dotrun](/HACKING.md#maas-ui-development-setup)
- [Building in a development MAAS](/HACKING.md#running-maas-ui-from-a-development-maas)

### QA steps

- Go to the machine storage tab and check that the data shown is identical to the angular version. That is, unmounted filesystems (disk, partition, and special) should show up here.

## Fixes

Fixes canonical-web-and-design/MAAS-squad#2156

## Screenshot
![Screenshot_2020-11-09 illegal-finch maas storage bolla MAAS](https://user-images.githubusercontent.com/25733845/98510790-73a8de00-22af-11eb-912f-14a943aa0902.png)
